cpp/core/utils/TaskContext.cc

```diff
@@ -40,6 +40,8 @@ class TaskContextStorage {
 };
 
 thread_local std::unique_ptr<TaskContextStorage> taskContextStorage = nullptr;
+
+// TODO: Remove this and let related unmanaged resources be managed.
```

Comment:
So we still have some memory unmanaged, right?

Reply:
Please ignore this obsolete comments and will clean later, I add false check when using fallback storage, ideally all native resources in under managed.